app/views/movies/show.html.erb

```diff
@@ -1 +1,29 @@
-<p>Hello World!</p>
\ No newline at end of file
+<%= button_to "Create A Viewing Party", new_user_movie_viewing_party_path(movie_id: @movie.show_movie.movie_id), method: :get, data: { turbo: false } %>  
+
+<br><%= button_to "Discover Movies", user_discover_index_path, method: :get, data: { turbo: false } %> 
+
+<h1><%= "#{@movie.show_movie.original_title}" %> Details</h1>
+
+<ul class="movie-details">
+  <li><%= @movie.show_movie.original_title %></li>
+  <li><%= @movie.show_movie.vote_average %></li>
+  <li><%= @movie.show_movie.runtime %></li>
+  <li><%= @movie.show_movie.all_genres %></li>
+  <li><%= @movie.show_movie.summary %></li>
+</ul>
```

Comment:
I see that we are using those methods that we built in our poros here so that it will show up on the webpage correctly.